src/main/java/com/codurance/retropolis/entities/User.java

```diff
@@ -8,4 +8,8 @@ public User(Long id, String email) {
     this.id = id;
     this.email = email;
   }
+
+  public Long getId() {
```

Comment:
Can you please remove the `getId()` here? The fields are `public final` so you can access the `id` directly